server/views/applications/pages/accommodation-need/contact-details/probation-practitioner.njk

```diff
@@ -1,12 +1,24 @@
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
 {% extends "../../layout.njk" %}
 
 {% block questions %}
+    {% set context = fetchContext() %}
+
+    <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l">{{ section.title }}</span>
+        {{ page.title }}
+    </h1>
+
+    <p>This information has come from nDelius.</p>
 
-  <h1 class="govuk-heading-l">
-    <span class="govuk-caption-l">{{ section.title }}</span>
-    {{ page.title }}
-  </h1>
+    {{ govukInsetText({
+        text: 'You need to update anything that is incorrect or missing.'
+    }) }}
 
-  {% include "../../partials/_contact-details.njk" %}
```

Comment:
I was wondering, if this partial was still needed so searched the code base to see its only mentioned once here https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/blob/main/server/views/applications/pages/accommodation-need/contact-details/backup-contact.njk#L10. If its only being used once then could we get rid of the partial and move its contents to the one place it is referenced.

Reply:
Oooh that's a good spot!